deploy/docs/Best_Practices.md

```diff
@@ -129,10 +129,23 @@ In the fluentbit overrides file (https://github.com/SumoLogic/sumologic-kubernet
 Reference: https://fluentbit.io/documentation/0.12/input/tail.html 
 
 ### Filtering Prometheus Metrics by Namespace in the Remote Write Config
-If you want to filter metrics by namespace, it can be done in the prometheus remote write config. Here is an example of excluding kube-state metrics.
+If you want to filter metrics by namespace, it can be done in the prometheus remote write config. Here is an example of excluding kube-state metrics for namespace1 and namespace2:
 ```bash
  - action: drop
    regex: kube-state-metrics;(namespace1|namespace2)
    sourceLabels: [job, namespace]
 ```
-The above section should be added in each of the  kube-state remote write blocks.
\ No newline at end of file
+The above section should be added in each of the  kube-state remote write blocks.
+
+Here is another example of excluding up metrics in the sumologic namespace and keep the up metrics for all other namespaces:
```

Comment:
suggest you change  "and keep the up metrics" to "while still collecting metrics"